project/default.properties

```diff
@@ -1,2 +0,0 @@
-name=akka-quickstart-scala
```

Comment:
Are we sure this is unused? The text is suspiciously the same as on https://developer.lightbend.com/start/?group=akka

Reply:
> Are we sure this is unused? 

I am not so sure, but I didn't find the same file at https://github.com/akka/akka-http-quickstart-scala.g8 and https://github.com/akka/akka-http-quickstart-java.g8. I also found this `default.properties ` and `src/main/g8/default.properties` shared almost the same property values. I think I can restore since there are considerations. 

BTW, I noticed the link icon of github of https://developer.lightbend.com/start/?group=akka&project=akka-http-quickstart-java and https://developer.lightbend.com/start/?group=akka&project=akka-http-quickstart-scala are both linking to `10.0.x` branch.